docs/_templates/class.rst

```diff
@@ -0,0 +1,7 @@
+{{ fullname | escape | underline}}
```

Comment:
May want to put the header in this file too, and possibly in the CSS file. @jmertic do you recommend also putting the license/copyright statements in the various logistical yml files above as well? I did in OCIO, but I'm not sure how necessary it is in these files (i.e. `continuous_integration.yml`)

Reply:
Yep - best practice is to add them as a comment if the filetype supports it.

Reply:
Added to last stragglers and forced-push!